### DISCUSSION TITLE
What do you guys want me to focus on at the moment?

### DISCUSSION THREAD

**Discussion by aflacc** [aflacc/ProjectFNF#48] [Development] [answered]
So if you haven't been caught up, I have a few things I wanted to add to the engine. But with so many things started but never finished, I think its time to have a proper poll. We will use Reactions to do this poll

## ❤️ Modcharts
Modcharts have been a thing I've loved for a while. They are really annoying to work with however. They caused some compiling issues in the past and they will eventually be a thing but are really complicated

## 🚀 Dialogue Revamp
Dialogue will be revamped (explained a lot more in #47), I am not going to re-write everything so just go look at that.

## 🎉 Discord RPC complete makeover
Basically what this would do is make ProjectFNF make discord RPC look completely unique from the base rich presence.

## 👀  Week 7
:trollface: lmao u thought

**Comment by aflacc** [comment]
first person to reply to this discussion post gets marked as answer

**Reply by kijikodev** [reply]
makeover of the discord rpc quack

**Reply by aflacc** [reply]
no not reply this message

**Reply by aflacc** [reply]
also you need to lcick the reaction

**Reply by kijikodev** [reply]
tf
my brain seems to not work

**Comment by ghost** [comment, marked as answer]
My selection reason is because I can't compile without this being changed away from Lua

**Reply by ghost** [reply]
Currently modcharts are actually working and I can compile.

**Comment by UniversePM** [comment]
my dms

**Reply by aflacc** [reply]
Yes I understand you want to do a mod. Again I am overwelmed by literally every one of the like 30 mods teamfnf put down as if im expected to do them all at once
